src/library.js

```diff
@@ -642,11 +642,30 @@ LibraryManager.library = {
 
   system__deps: ['__setErrNo'],
   system: function(command) {
+#if ENVIRONMENT_MAY_BE_NODE
+    if (ENVIRONMENT_IS_NODE) {
+      if (!command)
```

Comment:
please use `{`, `}` if the if body is on another line. In this case though maybe just move it to the same line.